.pnp.cjs

```diff
@@ -46001,9 +46001,17 @@ async function copyFolder(prelayout, postlayout, updateTime, destinationFs, dest
   let updated = false;
 
   if (destinationStat === null) {
-    prelayout.push(async () => destinationFs.mkdirPromise(destination, {
-      mode: sourceStat.mode
-    }));
+    prelayout.push(async () => {
```

Comment:
The build wanted me to `yarn install` but I don't understand why this change is here...

Reply:
The release workflow updated the file that this file is based on in https://github.com/yarnpkg/berry/commit/d7fecbed653462cf678ece387d8aba0f864ab6ab#diff-e2f3ae0b0dd7c014ad4e4fce4cb52742573bedb3761435d7f847570ab12730b9 but didn't commit it, the change to that function itself comes from https://github.com/yarnpkg/berry/pull/3338/files#diff-f9a18e543151537cdfdb81ff2877ba0d9c2c1c0876987e98c893b379455c2ec7

Reply:
Yeah, it looks like the master build is currently broken: https://github.com/yarnpkg/berry/runs/3595736129.